src/pages/news/index.vue

```diff
@@ -1,15 +1,44 @@
 <template>
 	<div>
-		<base-button design="primary" @click="$router.push({ name: 'news-new' })">
+		<base-button
+			class="create-news-btn"
+			design="primary"
+			@click="$router.push({ name: 'news-new' })"
+		>
 			Artikel anlegen
 		</base-button>
-		<section v-if="news && news.length > 0" class="section">
+
+		<div class="view-toggles">
+			<!-- TODO: Find correct Icons! and show correct one on active -->
+			<base-button design="primary icon text" @click="toDisplayStyle('grid')">
+				<base-icon source="material" icon="view_column" fill="gray" />
+			</base-button>
+			<base-button design="primary icon text" @click="toDisplayStyle('list')">
+				<!-- TODO: Change for correct icons -->
+				<base-icon
+					v-if="isList"
+					source="material"
+					icon="view_list"
+					fill="gray"
+				/>
+				<base-icon v-else source="material" icon="list" fill="gray" />
+			</base-button>
+		</div>
+		<section :class="{ 'grid-container': !isList, list: isList }">
 			<news-card
 				v-for="article of news"
+				:id="article._id"
 				:key="article._id"
-				:article="article"
-				class="mb--md"
-			/>
+				:category="article.category"
+				:title="article.title"
+				:created-at="article.createdAt"
+				:created-by="article.creator.firstName + ' ' + article.creator.lastName"
+				:picture="article.picture"
+				:event-date="article.eventDate"
+				:is-landscape="isList"
+				>{{ article.content }}</news-card
```

Comment:
News are actually not plain text. The content is html.
```suggestion
				v-html="article.content"
				/>
```
![Uploading image.png…]()